### PR TITLE
New commands and playbooks for SecurityScorecard

### PR DESCRIPTION
**Pull request by LHousehold**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [ ] Ready
- [x] In Hold - (Reason for hold)

## Description
The update includes three new playbooks and a few new commands to power them. This new content is able to interact with SecurityScorecard's Event Log, pulling in event data and then fetching detailed information for particularly relevant events. Furthermore it is able to bring in metadata for issues such as description and resolution information.

### CODE REVIEW

**Comment by kgal-pan** [comment]
@LHousehold - For when filling out the [Contribution Form](https://forms.gle/XDfxU4E61ZwEESSMA), your Partner ID is `securityscorecard-632`.

**Comment by kgal-pan** [comment]
The _Run Unit Testing And Lint_ failures will be solved by https://github.com/demisto/content/pull/32232.

**Comment by kgal-pan** [comment]
@LHousehold - Seems that the PR has only 1 added Playbook while the PR description mentions the following:

> The update includes three new playbooks and a few new commands to power them.

Can you please add these changes to the PR as well?

**Comment by LHousehold** [comment]
> @LHousehold - Seems that the PR has only the unified integration while the PR description mentions the following:
> 
> > The update includes three new playbooks and a few new commands to power them.
> 
> Please separate the integration YAML by using:
> 
> ```shell
> demisto-sdk split --input Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml
> ```
> 
> And add these changes/new Playbooks to the PR.

I think the playbooks did not get included in my export. I added them manually in a way that imitates other packages.

**Comment by YuvHayun** [comment]
Hey @LHousehold, as per @kgal-pan suggestion.
Please separate the integration code & yml and let us know when you're ready for review.

File: Packs/SecurityScorecard/Integrations/SecurityScorecard/README.md
Comment (kgal-pan) [inline Packs/SecurityScorecard/Integrations/SecurityScorecard/README.md:1]: Please run `demisto-sdk generate-docs -i Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml` to add the new commands to the doc.

File: Packs/SecurityScorecard/Playbooks/SecurityScorecard_Event.yml
Comment (kgal-pan) [inline Packs/SecurityScorecard/Playbooks/SecurityScorecard_Event.yml:1]: Run `demisto-sdk generate-docs -i Packs/SecurityScorecard/Playbooks/SecurityScorecard_Event.yml`.

File: Packs/SecurityScorecard/Playbooks/SecurityScorecard_Events.yml
Comment (kgal-pan) [inline Packs/SecurityScorecard/Playbooks/SecurityScorecard_Events.yml:1]: Run `demisto-sdk generate-docs -i Packs/SecurityScorecard/Playbooks/SecurityScorecard_Events.yml`.

File: Packs/SecurityScorecard/Playbooks/SecurityScorecard_Sync.yml
Comment (kgal-pan) [inline Packs/SecurityScorecard/Playbooks/SecurityScorecard_Sync.yml:1]: Run `demisto-sdk generate-docs -i Packs/SecurityScorecard/Playbooks/SecurityScorecard_Sync.yml`.

**Comment by kgal-pan** [comment]
@LHousehold - Let's bump the version and fill out the release notes by running `demisto-sdk update-release-notes -i Packs/SecurityScorecard`

File: Packs/SecurityScorecard/ReleaseNotes/1_0_7.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+
+##### SecurityScorecard
+
+- %%UPDATE_RN%%
```
Comment (kgal-pan) [inline Packs/SecurityScorecard/ReleaseNotes/1_0_7.md:6]: Please fill this out with the added commands. See https://xsoar.pan.dev/docs/documentation/release-notes#enhancement-examples as an example.

**Comment by YuvHayun** [comment]
Closing this pr as it duplicates [this](https://github.com/demisto/content/pull/32437) pr.

**Pull request closed without merging**